ush/config.sh.RRFS_AK_dev2

```diff
@@ -0,0 +1,77 @@
+MACHINE="jet"
+ACCOUNT="nrtrr"
+EXPT_BASEDIR="/mnt/lfs4/BMC/rtwbl/mhu/rrfs/dev2"
```

Comment:
Please use the paths to the rtrr space here, and in ptmp/stmp below.

Reply:
Done